components/UserRating.js

```diff
@@ -0,0 +1,28 @@
+import { useOperation } from "react-openapi-client";
+import Rating from "@mui/material/Rating";
+
+function UserRating({ resourceId }) {
+  const { loading, data, error } = useOperation(
+    "getResourceEvaluation",
+    resourceId
+  );
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+  }
+
+  return (
+    <Rating
+      name="simple-controlled"
+      value={data.evaluation}
+      onChange={(event, newValue) => {
+        console.log(newValue);
```

Comment:
Also please remove the console.log

Reply:
I keep this console.log() because in the future I will replace it with a function to handle the OnChange action. By now, I need to provide a function, otherwise ESlint thrown an error.